proposals/NNNN-steering-wheel-location.md

```diff
@@ -0,0 +1,52 @@
+# Steering wheel location
+
+* Proposal: [NNNN](NNNN-steering-wheel-location.md)
+* Author: [Kujtim Shala](https://github.com/kshala-ford)
+* Status: **Review ready**
+* Impacted Platforms: [Core / iOS / Android / RPC ]
+
+## Introduction
+
+This proposal is about adding a new parameter to the `VehicleType` structure to inform the app about the steering wheel location within the vehicle.
+
+## Motivation
+
+Some apps want to use different templates based on the steering wheel location. As an example TEXT_WITH_BUTTONS_AND_GRAPHIC works great for left hand drives as buttons are closer to the driver. For right hand drives apps would want to use GRAPHIC_WITH_TEXT_AND_BUTTONS. Today apps would use the language/region code and **guess** the location of the steering wheel.
+
+## Proposed solution
+
+In order to improve the user experience for right hand drive the app should know the steering wheel location. This information can be provided to the app right after the apps registration on the head unit. This newly added parameter should be optional for backwards compatibility. The type of the parameter should be an enum. Following values should be allowed:
```

Comment:
Please change "right hand drive" to "right handed drivers," and "apps registration" to "app's registration"